control-plane/roles/isolated-clusters/templates/dns/coredns-deployment.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: coredns
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: coredns
+  name: coredns
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 2
+  selector:
+    matchLabels:
+      k8s-app: coredns
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        k8s-app: coredns
+    spec:
+      containers:
+      - args:
+        - -conf
+        - /etc/coredns/Corefile
+        env:
+        image: "{{ isolated_clusters_dns_image_name }}:{{ isolated_clusters_dns_image_tag }}"
+        imagePullPolicy: IfNotPresent
```

Comment:
There is a control plane default variable for pull policy, maybe would be nice to use it.